setup.py

```diff
@@ -28,6 +28,7 @@
 from setuptools import setup
 
 setup(name='python-libs',
+      version='3.0.0',
```

Comment:
Is the setup.py needed at all after pyproject.toml is present? I'd try to move dependencies to the toml and nuke this

Reply:
Done in PR 27!